pkg/wookie/wookie.go

```diff
@@ -17,36 +17,41 @@ package wookie
 import (
 	"context"
 	"net/http"
+
+	"github.com/pkg/errors"
 )
 
-const HeaderName = "Warrant-Token"
+const WarrantTokenHeaderName = "Warrant-Token"
```

Comment:
Can you change this back to `HeaderName` or is there a reason for the rename? With the full pkg name, it'll be `wookie.HeaderName` which should be descriptive enough.